### ISSUE TITLE
Music site

### ISSUE THREAD

**Issue by susl16c** [mgks/Android-SmartWebView#17]
Tell me, if I build a music site in the application, when will I turn it off, will it play music?

**Comment by mgks** [comment]
you have to test it on your own. I've not tested any music website on it.

**Comment by susl16c** [comment]
Yes, I already checked, I need to add that the site worked as a service, then when folding the music will play. Now it stops playing when the application is minimized.

**Comment by tfstfs** [comment]
Great code! Thanks. I have a question. Can you customize your code for me so that the app runs in background and every 5 minutes sends a request to the server (together with webview cookies) to check for new messages? If there are new messages they must appear as notifications with sound y/o vibration and link to the app? If yes, tell me your price for this.

**Comment by mgks** [comment]
Hey.
I was just looking into this and this thing is a whole new work around that will take a lot of time.
Meanwhile, I can provide you with some ways which you can try yourself until this repo updates with audio features.

- First you need to add a separate/independent service that can run in the background even if your app is not in foreground.
- For controls, you need a separate method clause for your Media Player.
- If you need an instant solution, try adding sticky service to your app to keep the player in the foreground.

That's all I can do for now. You've to find a work around by yourself.

**Comment by susl16c** [comment]
Let's not close this problem until now, maybe someone will be able to respond and help us, in the future it can help other people and make the application more functional.
